MODULE.bazel

```diff
@@ -0,0 +1,31 @@
+module(
+    name = "rules_detekt",
+    version = "0.7.0",
+    compatibility_level = 1,
+    repo_name = "bazel_rules_detekt",
+)
+
+register_toolchains("//detekt:default_toolchain")
+
+detekt = use_extension("//detekt:extensions.bzl", "detekt")
+use_repo(detekt, "detekt_cli_all")
+
+bazel_dep(name = "rules_java", version = "5.4.1")
+bazel_dep(name = "rules_proto", version = "5.3.0-21.7")
+bazel_dep(name = "rules_jvm_external", version = "5.3")
+maven = use_extension("@rules_jvm_external//:extensions.bzl", "maven")
+maven.install(
+    name = "rules_detekt_dependencies",
+    artifacts = [
+        "com.squareup.moshi:moshi:1.14.0",
+        "com.squareup.okio:okio-jvm:3.2.0",
+        "io.reactivex.rxjava3:rxjava:3.0.12",
+        "junit:junit:4.13.2",
```

Comment:
JUnit needs to be a testonly artifact 

https://github.com/buildfoundation/bazel_rules_detekt/blob/ae20a19eb98ef1aceaed41932d61b3468454e758/detekt/toolchains.bzl#L35